### DISCUSSION TITLE
[Please Help]My Markdown table rendering incorrectly withou any syntax problem.

### DISCUSSION THREAD

**Discussion by Jessie-Create24** [iamcco/markdown-preview.nvim#694] [Q&A]
Here are the coding and results.
My Coding:
![Screenshot from 2024-08-17 13-04-16](https://github.com/user-attachments/assets/36b5f2d6-ba6f-4054-b7e6-8032d03810ed)
My Results:
![image](https://github.com/user-attachments/assets/3b157589-c3d4-4721-889b-8832afb52df5)

**Comment by Tweekism** [comment]
Have you tried replacing all the `+` with just `|`

**Comment by DrKJeff16** [comment]
Do they render in other environments?
_I'm no MD expert_, but I feel the inconsistent use of, `===` and `---` lines might be at fault here (**in particular lines `125` and `130`**)
Also, I agree with @Tweekism 's suggestion aswell

Could you paste the relevant (or whole, your call) code snippet instead of using pictures?
Would make it easier to replicate
